aws/resource_aws_spot_fleet_request.go

```diff
@@ -623,6 +633,24 @@ func resourceAwsSpotFleetRequestCreate(d *schema.ResourceData, meta interface{})
 		return err
 	}
 
+	if d.Get("wait_for_fulfillment").(bool) {
+		log.Println("[INFO] Waiting for Spot Fleet Request to be fulfilled")
+		spotStateConf := &resource.StateChangeConf{
+			Pending:    []string{"pending_fulfillment"},
+			Target:     []string{"fulfilled"},
+			Refresh:    resourceAwsSpotFleetRequestFulfillmentRefreshFunc(d, meta),
+			Timeout:    10 * time.Minute,
```

Comment:
The customizable timeout is currently ignored here - do you mind changing it to `d.Timeout(schema.TimeoutCreate)`? 😉